string[] objects
string target_object
---
bool succeeded